### ISSUE TITLE
resolv_conf

### ISSUE THREAD

**Issue by chunyingw** [bluejekyll/trust-dns#994]
Ben,
I notice that on Fuchsia it emits `ERROR    winutil (can't find repo path for winutil)` when it loads crates from crates.io. The reason it is loaded is because `trust-dns-resolver`->`resolv-conf`->`hostname(0.15.0)`->winutil.

resolv-conf crate is used for unix, so it won't be used on Fuchsia anyway. However it's ugly to see this error message. I don't understand why Fuchsia complains this even though it only tries to fetch the crate and not uses it. I will dig into this.

Meanwhile I'd like to hear your advice on how to solve it in different ways:
Option1. In trust-dns-resolver, I may add a feature like unix-resolver-conf which contains `resolv-conf` and it will be part of the default features, in this way Fuchsia won't try to fetch this crate at all, so the error message will be gone.

Option2. I see the latest version of `hostname 0.30.0` does not depend on wintuil, it seems if resolv-conf bumps its hostname dependency version from 0.15.0 to 0.30.0. It will work. I see `resolv-conf` was last updated one year ago, not sure if the owner is actively maintaining it.

The bottom line is with the PR#993 being part of the published version, I think I will be able to build  trust-dns-proto and trust-dns-resolver (grabbing from crates.io instead of patching) on Fuchsia successfully, but I will see this `ERROR    winutil (can't find repo path for winutil)` message when update the crates from crates.io.

**Comment by chunyingw** [comment]
update: On Fuchsia side, wintil is vendored(loaded) successfully, the above error is emit during license validation. winuil is MIT licensed, I think it's okay. It could just be how the checking script is written that cause the issue. I will look further.

**Comment by bluejekyll** [comment]
Ok. Ive just published 0.19.1. I’m open to options here.

**Comment by chunyingw** [comment]
Update: with the version 0.19.1, I am able to port `trust-dns-proto` and `trust-dns-resolver` onto Fuchsia. Hooray!

For winutil (verison 0.1.1), Fuchsia is looking for a file like 'LICNESE` or such to do the validation, and this file is missing from the repo (version 0.1.1). That's why I see that error message.

I will see if it's possible I update the script on Fuchsia and get approved first. Then I don't need to make changes to trust-dns-resolver. 
Otherwise, I will make resolv-conf optional.

**Comment by chunyingw** [comment]
After some research I think our script is doing the right thing. 
Essentially it's validating against `LICENSE` file and does not trust the `license` field in the `Cargo.toml` file. 
[`winutil`] (https://bitbucket.org/DaveLancaster/winutil/src/master/) is missing the LICENSE* file from both the published version and github repo.

So I will work on making `resolv-conf` optional for `trust-dns-resolver` now.
